modules/ROOT/partials/misc/plugin_support_table.adoc

```diff
@@ -20,6 +21,5 @@ The following list of premium plugins are supported by {companynameformal} for {
 * xref:introduction-to-powerpaste.adoc[PowerPaste]
 * xref:rtc-introduction.adoc[Real-Time Collaboration (RTC)]
 * xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro]
-* xref:premium-skins-and-icons.adoc[{prem_skins_icons}]
-* xref:editimage.adoc[EditImage]
-* xref:tableofcontents.adoc[TableOfContents]
\ No newline at end of file
+* xref:tableofcontents.adoc[TableOfContents]
```

Comment:
```suggestion
* xref:tableofcontents.adoc[Table of Contents]
```
This name is incorrect, it should be this.

Reply:
Just curious, are the name of MoxieManager and PowerPaste correct?

Reply:
Yes they are correct. Take a look at the actual plugin pages to see their name, or look at the product pages. For example: https://www.tiny.cloud/tinymce/features/powerpaste